.circleci/config.yml

```diff
@@ -82,8 +82,23 @@ jobs:
       - image: circleci/python:2.7
     working_directory: ~/answers
     steps:
+      - run:
+          name: "Setup branch formatting, replace / with -"
+          command: echo 'export FORMATTED_CIRCLE_BRANCH=$(echo ${CIRCLE_BRANCH} | sed "s/\//-/g")' >> $BASH_ENV
```

Comment:
could you explain to me what sed does here exactly? my only experience with sed is somebody implementing snake with it (l o l), reading up on some docs right now

Reply:
is this replacing all forward slashes with dashes?

Reply:
sure thing, sed is editing whatever is input to it
`s` is the replace command, delimits params with the `/`
so it's searching for `\/` which is an escaped `/` and replacing it with `-`
the `g` flag makes it global, looking everywhere in the stream instead of just one place

that's from memory, correct me if im wrong anywhere